std/stdio.d

```diff
@@ -2125,8 +2128,9 @@ $(XREF file,readText)
         void test(Terminator)(string txt, in string[] witness,
                 KeepTerminator kt, Terminator term, bool popFirstLine = false)
         {
+            import std.array : array;
```

Comment:
This was being pulled in by a public import inside the import of `std.range` above, got rid of that import and made this explicit here.